index.js

```diff
@@ -118,6 +125,11 @@ class ResembleHelper extends Helper {
      * @returns {Promise<void>}
      */
     async seeVisualDiffForElement(selector, baseImage, options){
+        const outputFile = screenshotOutputFolder(baseImage);
+        this.debug(`Screenshot is saving to ${outputFile}`);
+        const browser = this._getBrowser();
+        await browser.saveScreenshot(outputFile);
```

Comment:
you could actually use the helper function of CodeceptJS, `I.saveScreenshot` here, this also seems fine.